Wikimate.php

```diff
@@ -226,22 +235,30 @@ private function request($data, $headers = array(), $post = false)
 
 	/**
 	 * Obtains a wiki token for logging in or data-modifying actions.
+	 *
+	 * If a CSRF (default) token is requested, it is stored and returned
+	 * upon further such requests, instead of making another API call.
 	 * For now this method, in Wikimate tradition, is kept simple and supports
 	 * only the two token types needed elsewhere in the library.  It also
 	 * doesn't support the option to request multiple tokens at once.
 	 * See {@see https://www.mediawiki.org/wiki/Special:MyLanguage/API:Tokens}
 	 * for more information.
 	 *
 	 * @param   string  $type  The token type
-	 * @return  string         The requested token
+	 * @return  mixed          The requested token (string), or null if error
```

Comment:
Just a small nitpick: "null if error" → "null in case of error".

Reply:
Hmm, there are five more places using the same phrase. Either all are changed, or kept as is. I think it's fine as is.

Reply:
Yeah, makes sense. Let's leave it as is.